apps/docs/src/app/core/component-docs/date-picker/examples/date-picker-form-example.component.ts

```diff
@@ -4,23 +4,34 @@ import { FdDate } from '@fundamental-ngx/core';
 
 @Component({
     selector: 'fd-date-picker-form-example',
-    styles: [`
-        .flex-form{
-            display:flex;
-            justify-content: space-between;
-        }
-    `],
     template: `
         <form [formGroup]="customForm" class="flex-form">
             <div>
-                <fd-date-picker formControlName="date"></fd-date-picker><br/>
+
+
+                <div fd-form-item>
+                    <label fd-form-label>Date Picker</label>
+                    <fd-date-picker [state]="'valid'" formControlName="date"></fd-date-picker>
```

Comment:
This shouldn't be a constant being passed to [state] because if the user puts a false one it should not be seen as 'valid'. I think maybe instead it could be a function that returns true or false depending on the format or something?

Reply:
I changed this example, now it has dynamic state `valid` or `inavlid`